Gems/ImGui/Code/Source/ImGuiManager.cpp

```diff
@@ -246,6 +247,17 @@ void ImGuiManager::Render()
 {
     if (m_clientMenuBarState == DisplayState::Hidden && m_editorWindowState == DisplayState::Hidden)
     {
+        // the first frame that this is true means that it has been deactivated, the following condtional is to avoid
+        // continuous bus notifications
+        if (m_ImGuiBroadcastState.m_deactivationBroadcasted == ImGuiActivationBroadcast::DeactivationNotBroadcasted)
+        {
+            AzFramework::ViewportImGuiNotificationBus::Broadcast(&AzFramework::ViewportImGuiNotificationBus::Events::OnImGuiDeactivated);
+            m_ImGuiBroadcastState.m_deactivationBroadcasted = ImGuiActivationBroadcast::DeactivationBroadcasted;
+
+            // reset the following state to ActivationNotBroadcasted so when ImGui is first activated,
+            // it sends the activation bus notification
+            m_ImGuiBroadcastState.m_activationBroadcasted = ImGuiActivationBroadcast::ActivationNotBroadcasted;
```

Comment:
suggestion: Please can we drop the `ed` after Broadcast in the enum names, it's not required and without arguable reads more clearly